packages/shared/src/hooks/useUserMention.ts

```diff
@@ -213,7 +213,7 @@ export function useUserMention({
   };
 
   const onInitializeMentionButtonClick = () => {
-    if (typeof query !== 'undefined') {
+    if (typeof query !== 'undefined' || !commentRef.current) {
```

Comment:
I think we should still have this in case the ref for comments is not available right?
What happens when you simply type `@T` (without the button)

Reply:
I tested this and it seems to work fine :)